src/mesh/mesh.f90

```diff
@@ -247,6 +247,25 @@ SUBROUTINE InitMesh()
       CALL abort(__STAMP__,&
         'Specified curving method does not exist. =1: NormalVectors, =3: SplitElemFile, =4: SpecElemFile')
     END SELECT
+
+    ! Volume curving by radial basis functions
+    useRBF = GETLOGICAL('useRBF','.FALSE.')
+    IF (useRBF) THEN
+      nRBFBoxes = CNTSTR('RBFType','0')
+
+      ALLOCATE(SupportRadius(nRBFBoxes))
+      ALLOCATE(RBFType(      nRBFBoxes))
+      ALLOCATE(xlim(       2,nRBFBoxes))
+      ALLOCATE(ylim(       2,nRBFBoxes))
+
+      DO i=1,nRBFBoxes
+        SupportRadius(i) = GETREAL('SupportRadius')
+        RBFType(i) = GETINT('RBFType')
+        xlim(:, i) = GETREALARRAY('xlim',2,'-1000000,1000000')
+        ylim(:, i) = GETREALARRAY('ylim',2,'-1000000,1000000')
```

Comment:
Please write `HUGE(1)` here

Reply:
Is implemeted. Please resolve if sufficient.